single_line_class.rb

```diff
@@ -0,0 +1,27 @@
+#Prefer a single-line format for class definitions with no body
+class FooError < StandardError
+end
+
+class Test
+	# Avoid single-line methods.
+    def too_much; puts 'pravin'; puts 'mishra' end 
+    
+    # bad
+    e = M * c ** 2
+    [ 1, 2, 3 ].size
```

Comment:
__[rubocop]__
* Do not compute the size of statically sized objects.
* Space inside square brackets detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-spaces-braces)
* Space inside square brackets detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-spaces-braces)